test/constructRoutes.test.js

```diff
@@ -704,4 +704,21 @@ describe("constructRoutes", () => {
       );
     });
   });
+
+  describe("router config defined in HTML", () => {
```

Comment:
Could we add a test verifying that `containerEl` is `"body"` when it's not defined in the HTML? We could probably reuse an old fixture and test for this, since they don't define the containerEl.